src/Search/Api/Elasticsearch/ElasticsearchClient.php

```diff
@@ -18,6 +18,12 @@ public function __construct(Client $connection, string $index, bool $forceSync =
         $this->forceSync = $forceSync;
     }
 
+    public function setIndex($indexName){
+
+        $this->index = $indexName;
```

Comment:
would rename this in `$this->defaultIndex` as it seems this class will be able to work with more than one at the same time, when `$index` is passed as a parameter to its methods.